### DISCUSSION TITLE
文档中点击实体控制台会报错

### DISCUSSION THREAD

**Discussion by ChauPha** [zouyaoji/vue-cesium#202] [Q&A]
在文档中点击实体控制台会报错，本地也是如此 之前没有这种情况 不知道怎么回事 还有label背景色变得很奇怪
![1652154123(1)](https://user-images.githubusercontent.com/24471468/167538340-07136714-7810-4d6a-81ce-543d6f9edb75.jpg)
![1652154164(1)](https://user-images.githubusercontent.com/24471468/167538374-3394283a-5740-4b7b-89ae-75b16fbe2b3a.jpg)

**Comment by zouyaoji** [comment]
https://zouyaoji.top/vue-cesium/#/zh-CN/component/datasources/vc-datasource-custom

我看这儿没问题呢，以这儿的为准呢，gitee 那个上次貌似同步失败了，看下个版本恢复正常不。
我本地也没重现，希望能提供复现方式。

**Reply by webdq** [reply]
https://zouyaoji.top/vue-cesium/#/zh-CN/component/primitives/vc-collection-primitive
@zouyaoji   点击图元集合也会报错，Cesium没有when方法了，cesium版本1.93.0

[![Oa4udO.png](https://s1.ax1x.com/2022/05/11/Oa4udO.png)](https://imgtu.com/i/Oa4udO)

[![Oa4lJH.png](https://s1.ax1x.com/2022/05/11/Oa4lJH.png)](https://imgtu.com/i/Oa4lJH)

**Reply by zouyaoji** [reply]
好的，这应该就是bug，将在下个版本解决。 请你暂时先用Cesium1.92以下版本吧。
